docs/rule/no-meta-redirect-with-time-limit.md

```diff
@@ -1,30 +0,0 @@
-## (no-meta-redirect-with-time-limit)
```

Comment:
Since we have released this one, we can't remove it here. Can you add an entry to the deprecated rules list for it?

See https://github.com/ember-template-lint/ember-template-lint/commit/2016af3b8698fe28d931a667c2a3bb34a7920a6a for an example.